tests/integ/modin/crosstab/test_crosstab.py

```diff
@@ -844,6 +844,172 @@ def eval_func(lib):
                 eval_func,
             )
 
+    @pytest.mark.parametrize("aggfunc", ["count", "mean", "min", "max", "sum"])
+    def test_margins_and_values(self, dropna, aggfunc):
+        a = np.array(
+            [
+                "foo",
```

Comment:
I think we are using this array ~10 times in this file. Can we make `a` a fixture (note we prefer fixtures since we don't want test cases to mutate them)?

Also can we reduce duplication for other highly reused inputs?